### PR TITLE
Feature/invoices permissions module

### PR DESCRIPTION
**Pull request by srivastava-jyoti**
Targets #1111

### CODE REVIEW

File: resources/views/layouts/navbar.blade.php [hunk 1]
```diff
@@ -51,18 +51,19 @@
...
 
-    @if(Module::checkStatus('Invoice') || Module::checkStatus('LegalDocument'))
+    @if((Module::checkStatus('Invoice') && auth()->user()->can('finance_invoices.view')) || (Module::checkStatus('LegalDocument') && auth()->user()->can('legal-document.view') ))
         <li class="nav-item dropdown">
             <a id="navbarDropdown_finance" class="nav-link dropdown-toggle" href="#" role="button"
                 data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" v-pre>Finance <span class="caret"></span>
             </a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown_finance">
-                @if(Module::checkStatus('Invoice') && auth()->user()->can('invoice.view'))
+                @can('finance_reports.view')
```
Comment (GauravGusain98) [inline resources/views/layouts/navbar.blade.php:60]: Is there any reason you removed `invoice.view` permission check and added this permission.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments.

File: resources/views/layouts/navbar.blade.php (hunk 1 above)
Comment (srivastava-jyoti) [reply resources/views/layouts/navbar.blade.php]: It was done by mistake.

**Review by GauravGusain98 — APPROVED** [review]
Looks good on code.

**Pull request merged**